docs/user_guide/assessment/index.rst

```diff
@@ -11,6 +11,17 @@ fairness metrics, such as demographic parity and equalized odds.
 We will show how :class:`MetricFrame` can be used to evaluate the metrics
 identified during the course of a fairness assessment.
 
+Fairlean provides two primary ways of assessing fairness: :class:`MetricFrame`,
+which can be used to perform disaggregated analysis of a particular performance 
+metric (such as accuracy, false positive rate, etc.) across sensitive 
+groups, and a set of predefined fairness metrics that use :class:`MetricFrame` 
+internally to output an aggregate number.
+:class:`MetricFrame` can also be used to output an aggregate number, 
```

Comment:
Nitpick: I'd prefer 'value' to 'number' here.

Reply:
Done